components/applications-service/pkg/params/sorting.go

```diff
@@ -27,6 +27,7 @@ var (
 	validSortFieldsForServiceGroups = map[string]SortField{
 		"name":        NameField,
 		"percent_ok":  PercentOkField,
+		"health":      HealthField,
```

Comment:
I don't think we added a sorting by health, based on the UI my understanding of the card is that it is sorting by percent_ok and then by health within like percentages.

Reply:
See comment above ☝️